### PR TITLE
Feature/overhaul message formatting

### PR DESCRIPTION
**Pull request by xmadsen**
**Merge this after the autopep8 branch** - all messages now come with a Header block depending on context, and are posted as a list to the 'blocks' parameter rather than as a str to 'message'. Also modified instances of sample commands in messages to use preformatted code formatting to differentiate the text better.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Python 3
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.6
```
Comment (jacksonal) [inline .github/workflows/main.yml]: app runtime is python 3.8
```suggestion
          python-version: 3.8
```

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
```
Comment (jacksonal) [inline .github/workflows/main.yml]: this should be requirements-dev.txt
```suggestion
          pip install -r requirements-dev.txt
```

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
+name: Run Python Tests
```
Comment (jacksonal) [inline .github/workflows/main.yml:1]: I have to ask... is this going to rack up a bill with micro$oft?

**Review by jacksonal — APPROVED** [review]
I tested some of these out in the block kit builder. very cool!

**Review by jacksonal — CHANGES_REQUESTED** [review]
see comments

File: .github/workflows/main.yml (hunk 3 above)
Comment (xmadsen) [reply .github/workflows/main.yml:1]: Not with our usage :)

**Review by jacksonal — APPROVED** [review]

**Pull request merged**